pr-data.csv

```diff
@@ -3768,7 +3768,7 @@ https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,
 https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,incubator/cdi-inject-weld,org.glassfish.jersey.inject.weld.internal.managed.DisposableSupplierTest.testDisposeComposedObjectWithPerLookupFields,ID,Opened,https://github.com/eclipse-ee4j/jersey/pull/5764,
 https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,incubator/cdi-inject-weld,org.glassfish.jersey.inject.weld.internal.managed.SupplierContractsTest.testClassFactoryMultipleContractsSingleton,ID,,,
 https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,media/json-jackson,org.glassfish.jersey.jackson.internal.DefaultJsonJacksonProviderForBothModulesTest.testDisabledModule,ID,Opened,https://github.com/eclipse-ee4j/jersey/pull/5768,
-https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,incubator/declarative-linking,org.glassfish.jersey.linking.integration.LinkingTest.orderContainsProvidedLinks,ID,,,
+https://github.com/eclipse-ee4j/jersey,8105415f9a6356de774c8ffe6ab5dfa77934b69f,incubator/declarative-linking,org.glassfish.jersey.linking.integration.LinkingTest.orderContainsProvidedLinks,ID,DeveloperFixed,,
```

Comment:
If you say that this test was not flaky in the reported commit, then the row should be removed. But, rather than running only one test method in isolation, did you try to run the entire test class or the entire test module? Did you run with `-DnondexRuns=10` or higher? Do you have logs that Ajitesh can inspect?

Reply:
I've tried running nondex on the test module, and 100 runs on the specific test. Where do I put the logs?

Reply:
`scp` the logs to the VM and let Ajitesh know to inpsect them.